danielMunteanu/src/main/java/IOtask/Utils.java

```diff
@@ -0,0 +1,37 @@
+package IOtask;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+
+public class Utils {
+    public String readFile(String filePath, String word) {
+        int wordCntr = 0;
+        String result = "";
+        boolean check;
+        try {
+            FileInputStream fstream = new FileInputStream(filePath);
+            BufferedReader br = new BufferedReader(new InputStreamReader(fstream));
+            String strLine;
+            while ((strLine = br.readLine()) != null) {
+                check = strLine.toLowerCase().contains(word.toLowerCase());
+                if (check) {
+                    String[] linewords = word.split("\\n+");
+                    for (String w : linewords) {
+                        {
+                            String foundWord = w.trim();
+                            if (word.equalsIgnoreCase(foundWord)) {
+                                wordCntr++;
+                            }
+                        }
+                    }
+                    result = "word " + word + " was found " + wordCntr + " times";
+                }
+            }
+            br.close();
```

Comment:
can we define this row in the finally block?

Reply:
defined